Dockerfile.redhat

```diff
@@ -143,10 +133,11 @@ ARG NVIDIA=0
 # Add Nvidia dev tool if needed
 # hadolint ignore=DL3003
 RUN if [ "$NVIDIA" == "1" ] ; then true ; else exit 0 ; fi ; \
-    yum config-manager --save --set-enabled codeready-builder-for-rhel-8-x86_64-rpms && \
+    yum config-manager --save --set-enabled codeready-builder-for-rhel-8-x86_64-rpms ; \
```

Comment:
Why we change '&&'-> ';'

Is there scenario where we need explicitly ignore errors in `yum` command?

Reply:
On a system without entitlements this repository will fail but it will not be used